packages/grafana-ui/src/components/Tabs/TabContent.tsx

```diff
@@ -10,7 +10,9 @@ interface Props {
 const getTabContentStyle = stylesFactory((theme: GrafanaTheme) => {
   return {
     tabContent: css`
-      padding: ${theme.spacing.xs} 0;
+      padding: ${theme.spacing.xs};
+      height: 90%;
```

Comment:
Stopping this at 90% otherwise the table reaches the bottom of the browser window which looks a bit weird in my opinion.